aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -184,11 +194,31 @@ mod test {
         aggregator_data.create_new_aggregator(aggregator_id_for_test(300), 300);
         aggregator_data.create_new_aggregator(aggregator_id_for_test(400), 400);
 
-        aggregator_data.get_aggregator(aggregator_id_for_test(100), 100);
-        aggregator_data.get_aggregator(aggregator_id_for_test(200), 200);
-        aggregator_data.get_aggregator(aggregator_id_for_test(500), 500);
-        aggregator_data.get_aggregator(aggregator_id_for_test(600), 600);
-        aggregator_data.get_aggregator(aggregator_id_for_test(700), 700);
+        aggregator_data.get_aggregator(aggregator_id_for_test(100), 100, context.resolver, true);
+        aggregator_data.get_aggregator(aggregator_id_for_test(200), 200, context.resolver, true);
+        aggregator_data.get_aggregator(aggregator_id_for_test(500), 500, context.resolver, true);
+        aggregator_data.get_aggregator(aggregator_id_for_test(600), 600, context.resolver, true);
+        aggregator_data.get_aggregator(aggregator_id_for_test(700), 700, context.resolver, true);
+
+        aggregator_data.remove_aggregator(aggregator_id_for_test(100));
+        aggregator_data.remove_aggregator(aggregator_id_for_test(300));
+        aggregator_data.remove_aggregator(aggregator_id_for_test(500));
+        aggregator_data.remove_aggregator(aggregator_id_for_test(800));
+    }
+
+    fn test_set_up_aggregator_disabled(context: &NativeAggregatorContext) {
```

Comment:
nit: Maybe let's pass a flag to set up function to avoid duplication?

Reply:
Done.